Doc/library/socket.rst

```diff
@@ -501,6 +503,20 @@ Constants
    .. availability:: Linux >= 2.2.
 
 
+.. data:: ETH_P_ALL
+
+   ``ETH_P_ALL`` can be used in :class:`~socket.socket`
+   constructor as *proto* for the :const:`AF_PACKET` family in order to
+   capture every packet.
```

Comment:
```suggestion
   :data:`!ETH_P_ALL` can be used in the :class:`~socket.socket`
   constructor as *proto* for the :const:`AF_PACKET` family in order to
   capture every packet, regardless of protocol.
```

* Use consistent syntax and semantics for ``ETH_P_ALL`` (without a link)
* Fix grammar
* Clarify the "every" dimension over which it operates (protocol, not host/dest address, port, interface, etc)